src/common/base/src/mem_allocator/mmap_allocator.rs

```diff
@@ -0,0 +1,304 @@
+// Copyright 2021 Datafuse Labs.
```

Comment:
```suggestion
// Copyright 2022 Datafuse Labs.
```

Reply:
Do we need another pr to update all the copyrights?

Reply:
Up to you. Both LGTM.

Reply:
> Do we need another pr to update all the copyrights?

No.
The year of the copyright means the file which year was created.